src/modalities/models/utils.py

```diff
@@ -0,0 +1,28 @@
+from pydantic import BaseModel
+from enum import Enum
+
+from modalities.config.component_factory import ComponentFactory
+from modalities.config.pydanctic_if_types import PydanticPytorchModuleType
+from modalities.registry.components import COMPONENTS
+from modalities.registry.registry import Registry
+
+class ModelTypeEnum(Enum):
+    MODEL = "model"
+    CHECKPOINTED_MODEL = "checkpointed_model"
+    
+def get_model_from_config(config: dict, model_type: ModelTypeEnum):
```

Comment:
`dict `is not a type annotation. Please use `Dict` from typing and the type annotations for key an value.